benchmark/fs/bench-chownSync.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+const tmpdir = require('../../test/common/tmpdir');
+tmpdir.refresh();
+
+if (process.platform === 'win32') {
+  // Windows does not have `getuid` or `getgid`
+  process.exit(0);
+}
+
+const tmpfile = tmpdir.resolve(`.existing-file-${process.pid}`);
+fs.writeFileSync(tmpfile, 'this-is-for-a-benchmark', 'utf8');
+
+const bench = common.createBenchmark(main, {
+  type: ['existing', 'non-existing'],
+  method: ['chownSync', 'lchownSync'],
+  n: [1e4],
+});
+
+function main({ n, type, method }) {
+  const uid = process.getuid();
+  const gid = process.getgid();
+  let path;
+
+  switch (type) {
+    case 'existing':
+      path = tmpfile;
+      break;
+    case 'non-existing':
+      path = tmpdir.resolve(`.non-existing-file-${Date.now()}`);
+      break;
+    default:
+      new Error('Invalid type');
+  }
+
+  bench.start();
+  for (let i = 0; i < n; i++) {
+    try {
+      fs[method](path, uid, gid);
```

Comment:
Might want to do the method lookup outside of the benchmark loop.

Reply:
👍